### DISCUSSION TITLE
How do you copy a Config Key on a Mac

### DISCUSSION THREAD

**Discussion by alejasper** [SafeExamBrowser/seb-mac#222] [Q&A] [answered]
On Windows, you can copy a configuration key as normal. However, on a mac all other applications are closed. You can copy but the clipboard is emptied as soon as you close down. There is an option to copy the Browser Exam Key, but not the Config Key. Am I missing something?

**Comment by danschlet** [comment, marked as answer]
The Config Key ideally is calculated by the exam system/LMS. If you need to copy/paste it, [start SEB with holding the option/alt key](https://safeexambrowser.org/macosx/mac_usermanual_en.html#configuration) (and optionally drag-and-drop a configuration file onto the app icon, SEB will ask for the administrator password in that file). Then the Preferences window is opened directly and SEB doesn't lock the computer, so you can copy the Config Key.

**Reply by alejasper** [reply]
Thanks.

**Reply by danschlet** [reply]
There was also an issue in SEB 3.0 for copy-pasting the BEK/CK keys, which was fixed in SEB 3.1.
